Documentation/gettingstarted/cni-chaining-azure-cni.rst

```diff
@@ -8,34 +8,123 @@
 Azure CNI
 *********
 
-This guide explains how to set up Cilium in combination with Azure CNI. In this
-hybrid mode, the Azure CNI plugin is responsible for setting up the virtual
-network devices as well as address allocation (IPAM). After the initial
-networking is setup, the Cilium CNI plugin is called to attach eBPF programs to
-the network devices set up by Azure CNI to enforce network policies, perform
-load-balancing, and encryption.
-
 .. note::
 
-   If you are looking to install Cilium on Azure AKS, see the guide
-   :ref:`k8s_install_aks` for a complete guide also covering cluster setup.
+   This is not the best option to run Cilium on AKS or Azure. Please refer to
+   :ref:`k8s_install_aks` for the best guide to run Cilium in Azure Cloud.
+   Follow this guide if you specifically want to run Cilium in combination with
+   the Azure CNI in a chaining configuration.
+
+This guide explains how to set up Cilium in combination with Azure CNI in a
+chaining configuration. In this hybrid mode, the Azure CNI plugin is
+responsible for setting up the virtual network devices as well as address
+allocation (IPAM). After the initial networking is setup, the Cilium CNI plugin
+is called to attach eBPF programs to the network devices set up by Azure CNI to
+enforce network policies, perform load-balancing, and encryption.
+
+
+Create an AKS + Cilium CNI configuration
+========================================
+
+Create a ``chaining.yaml`` file based on the following template to specify the
+desired CNI chaining configuration:
```

Comment:
Is it worth adding a sentence or two describing what `chaining.yaml` does? For example, the need for the `azure-vnet` and `ciluim` plugins is clear, but why is the `portmap` plugin needed?